examples/trials/ir_enas/trial_code/src/cifar10/general_child.py

```diff
@@ -0,0 +1,758 @@
+            pooled_layers.append(x)
+          return pooled_layers, out_filters
+
+
+      def post_process_out(layer_id, out, input_candidates):
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope("skip"):
+            inputs = layers[-1]
+            if self.data_format == "NHWC":
+              inp_h = inputs.get_shape()[1].value
+              inp_w = inputs.get_shape()[2].value
+              inp_c = inputs.get_shape()[3].value
+              out.set_shape([None, inp_h, inp_w, out_filters])
+            elif self.data_format == "NCHW":
+              inp_c = inputs.get_shape()[1].value
+              inp_h = inputs.get_shape()[2].value
+              inp_w = inputs.get_shape()[3].value
+              out.set_shape([None, out_filters, inp_h, inp_w])
+            if self.mode == 'subgraph':
+              input_candidates.append(out)
+              pout = tf.add_n(input_candidates)
+            else:
+              try:
+                pout = tf.add_n([out, tf.reduce_sum(input_candidates, axis=0)])
+              except Exception as e:
+                print(e)
+                pout = out
+            out = batch_norm(pout, is_training, data_format=self.data_format)
+
+        layers.append(out)
+
+        return out
+
+      def conv3(layer_id, res_layers):
+        # res_layers is pre_layers that are chosen to form skip connection
+        # layers[-1] is always the latest input
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope('branch_0'):
+            out = self._conv_branch(layers[-1], 3, is_training, out_filters, out_filters, start_idx=0)
+        return out
+      def conv3_sep(layer_id, res_layers):
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope('branch_1'):
+            out = self._conv_branch(layers[-1], 3, is_training, out_filters, out_filters, start_idx=0, separable=True)
+        return out
+      def conv5(layer_id, res_layers):
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope('branch_2'):
+            out = self._conv_branch(layers[-1], 3, is_training, out_filters, out_filters, start_idx=0)
+        return out
+      def conv5_sep(layer_id, res_layers):
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope('branch_3'):
+            out = self._conv_branch(layers[-1], 3, is_training, out_filters, out_filters, start_idx=0, separable=True)
+        return out
+      def avg_pool(layer_id, res_layers):
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope('branch_4'):
+            out = self._pool_branch(layers[-1], is_training, out_filters, "avg", start_idx=0)
+        return out
+      def max_pool(layer_id, res_layers):
+        with tf.variable_scope(layer_id):
+          with tf.variable_scope('branch_5'):
+            out = self._pool_branch(layers[-1], is_training, out_filters, "max", start_idx=0)
+        return out
+
+      """@nni.architecture
```

Comment:
cifar10 enas example?
 I think it's better to place a more simple example here.